migrations/v2.js

```diff
@@ -0,0 +1,464 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, getComponents, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('adapt-contrib-slider - v2.0.0 > v2.0.1', async () => {
+  let sliders;
+
+  whereFromPlugin('adapt-contrib-slider - from v2.0.0', { name: 'adapt-contrib-slider', version: '>=2.0.0 <2.0.1' });
+
+  whereContent('adapt-contrib-slider - where slider', async content => {
+    sliders = getComponents('slider');
+    return sliders.length;
+  });
+
+  mutateContent('adapt-contrib-slider - add slider._showScale', async () => {
+    sliders.forEach(slider => {
+      slider._showScale = true;
+    });
+    return true;
+  });
+
+  mutateContent('adapt-contrib-slider - add slider._recordInteraction', async () => {
+    sliders.forEach(slider => {
+      slider._recordInteraction = true;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-slider - check slider._showScale attribute', async () => {
+    const isValid = sliders.some(({ _showScale }) => _showScale === true);
+    if (!isValid) throw new Error('adapt-contrib-slider - _showScale not added to every instance of slider');
+    return true;
+  });
+
+  checkContent('adapt-contrib-slider - check slider._recordInteraction attribute', async () => {
+    const isValid = sliders.every(({ _recordInteraction }) => _recordInteraction === true);
+    if (!isValid) throw new Error('adapt-contrib-slider - _recordInteraction not added to every instance of slider');
+    return true;
+  });
+
+  updatePlugin('adapt-contrib-slider - update to v2.0.1', { name: 'adapt-contrib-slider', version: '2.0.1', framework: '>=2.0.0' });
+
+  testSuccessWhere('slider component with empty course', {
+    fromPlugins: [{ name: 'adapt-contrib-slider', version: '2.0.0' }],
+    content: [
+      { _id: 'c-100', _component: 'slider' },
+      { _type: 'course' }
+    ]
+  });
+
+  testStopWhere('no slider components', {
+    fromPlugins: [{ name: 'adapt-contrib-slider', version: '2.0.0' }],
+    content: [{ _component: 'other' }]
+  });
+
+  testStopWhere('incorrect version', {
+    fromPlugins: [{ name: 'adapt-contrib-slider', version: '2.0.1' }]
+  });
+});
+
+describe('adapt-contrib-slider - v2.0.1 > v2.0.3', async () => {
+  let sliders;
+
+  whereFromPlugin('adapt-contrib-slider - from v2.0.1', { name: 'adapt-contrib-slider', version: '<2.0.3' });
+
+  whereContent('adapt-contrib-slider - where slider', async content => {
+    sliders = getComponents('slider');
+    return sliders.length;
+  });
+
+  mutateContent('adapt-contrib-slider - add slider._correctAnswer', async () => {
+    sliders.forEach(slider => {
+      const newValue = slider._correctAnswer?.toString() || '';
```

Comment:
This looks to be an explicit conversion to the string data type. I assume it went from number to string at some point?

You can make this clearer to read by doing the type conversion at the end of the evaluation.

```suggestion
      const newValue = String(slider._correctAnswer ?? '');
```